script.js

```diff
@@ -1,4 +1,114 @@
+//Project 2: Beat That (Base+Comfortable)
+
+var rollDice = "ROLL DICE";
+var chooseSequence = "CHOOSE SEQUENCE";
```

Comment:
As a convention, constant variables like this that will not change value throughout the whole program can be named with `SCREAMING_SNAKE_CASE` i.e. `ROLL_DICE` and `CHOOSE_SEQUENCE`